config/locales/diaspora/en.yml

```diff
@@ -1091,18 +1091,23 @@ en:
 
   registrations:
     new:
+      title: "Join the community!"
+      subtitle: "You're about to create an account on %{podname}. One diaspora* account is enough to reach the whole network, you don't need to create a new account here if you already have one on another pod."
       enter_email: "Enter your email address"
-      enter_username: "Pick a username (only letters, numbers, and underscores)"
+      enter_username: "Pick a username (only letters, numbers, _ and -)"
       enter_password: "Enter a password (six character minimum)"
       enter_password_again: "Enter the same password as before"
       sign_up: "Create account"
       email: "Email"
+      email_advice: "Your e-mail will never be shared with anyone and will only be used to allow you to recover your account or send notifications."
       username: "Username"
+      username_advice: "Your username is unique and can't be changed. It allows others to find you in the network."
```

Comment:
"Your username is unique and allows others to find you in the network. It can't be changed once set."

Reply:
Better.